modular_skyrat/modules/tarkon/code/misc-fluff/research.dm

```diff
@@ -0,0 +1,122 @@
+///// First we enstate a techweb so we can add the node. /////
+
+/datum/techweb/tarkon
+	id = "TARKON"
+	organization = "Tarkon Industries"
+	should_generate_points = TRUE
+
+/datum/techweb/tarkon/New()
+	. = ..()
+	research_node_id("oldstation_surgery", TRUE, TRUE, FALSE)
+	research_node_id("tarkontech", TRUE, TRUE, FALSE)
+
+/datum/techweb_node/tarkon
+	id = "tarkontech"
+	display_name = "Tarkon Industries Technology"
+	description = "Tools used by Tarkon Industries."
+	prereq_ids = list("engineering")
+	required_items_to_unlock = list(
+		/obj/item/mod/construction/plating/tarkon,
+		/obj/item/construction/rcd/arcd/tarkon,
+		/obj/item/gun/energy/recharge/resonant_system,
+	)
+	design_ids = list(
+		"mod_plating_tarkon",
+		"arcs",
+		"rcd_tarkon"
+	)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 10000)
+	hidden = TRUE
+
+/datum/design/mod_plating/tarkon
+	name = "MOD Tarkon Plating"
+	id = "mod_plating_tarkon"
+	build_path = /obj/item/mod/construction/plating/security
```

Comment:
```suggestion
	build_path = /obj/item/mod/construction/plating/tarkon
```

Reply:
The build path is incorrect, should be `/obj/item/mod/construction/plating/tarkon`, or removed to avoid confusion.